prow/proxy-common.inc

```diff
@@ -68,4 +72,17 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
       export BAZEL_BUILD_ARGS="${BAZEL_BUILD_ARGS} --google_credentials=${GOOGLE_APPLICATION_CREDENTIALS} --remote_instance_name=${BAZEL_BUILD_RBE_INSTANCE} --remote_cache=${BAZEL_BUILD_RBE_CACHE}"
     fi
   fi
+elif curl -H "${METADATA_QUERY_HEADER}" "${METADATA_SERVER_URL}" | grep "${WORKLOAD_IDENTITY_SUFFIX}"; then
```

Comment:
i think it's bound to a GSA so may need to change the check to account for that? `gcloud auth list --filter=status:ACTIVE --format="value(account)"` may be simpler as well